ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -1297,3 +1298,83 @@ def test_advanced_funnel_multiple_exclusions_between_steps(self):
         self.assertCountEqual(
             self._get_people_at_step(filter, 1), [person4.uuid],
         )
+
+    def test_funnel_with_elements_chain(self):
```

Comment:
This is 3 separate tests in one where the whole filter is different each time. Why roll them into one?

Reply:
Hopefully the update makes more sense. It's a single test testing either element in the chain. Didn't make sense to split into two?